app-admin/keepassxc/keepassxc-9999.ebuild

```diff
@@ -39,7 +39,7 @@ RDEPEND="
 	dev-qt/qtnetwork:5
 	dev-qt/qtsvg:5
 	dev-qt/qtwidgets:5
-	dev-qt/qtx11extras:5
+	X? ( dev-qt/qtx11extras:5 )
```

Comment:
oh, also the conditional dependencies should be below normal ones,
so this should be before "autotype?"

Reply:
Sorry, corrected!